src/types/User.tsx

```diff
@@ -0,0 +1,6 @@
+export interface User {
+  id: number;
+  name: string;
+  username: string;
+  email: string;
```

Comment:
Consider using a more specific data type for email, like `Email` or `EmailAddress`. 📧